src/pages/Browse/index.jsx

```diff
@@ -234,7 +237,8 @@ const Browse = () => {
                     <input
                       type="search"
                       className="form-control"
-                      placeholder="Search"
+                      placeholder="Search in Page"
```

Comment:
Better name will be "Filter upload" IMO.

Reply:
Resolved